recipes/xorg/all/conanfile.py

```diff
@@ -76,8 +76,7 @@ def system_requirements(self):
                         "libxxf86vm", "libxv", "xkeyboard-config", "xcb-util"]
         if packages:
             package_tool = tools.SystemPackageTool(conanfile=self, default_mode="verify")
-            for p in packages:
-                package_tool.install(update=True, packages=p)
+            package_tool.install_packages(update=True, packages=packages)
```

Comment:
nice, I didn't know packages could be installed that way, without loop

Reply:
I just found out randomly this great piece of work by @uilianries https://github.com/conan-io/conan/pull/7779
That should make the install much faster

Reply:
:tada:

That one should be the new (and only) interface in the future, it is much better, easier to understand.

Reply:
I discovered it a while back, but was holding back with using it just yet, because of the need to raise the minimum Conan version, but I guess it is fine